cdc/sink/dispatcher/expression_test.go

```diff
@@ -0,0 +1,103 @@
+// Copyright 2022 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package dispatcher
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestParseTopicExpression(t *testing.T) {
+	// valid topic expressions containing '{schema}'
+	name, err := Parse("{schema}", "abc", "")
```

Comment:
It might be better to organize it as a test array, so that the strings have names(struct).